website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -7,6 +7,12 @@ page_id: api-changelog
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+### 2022-09-21
```

Comment:
Should this be the date of the public beta rather than when the documentation was added?